software/23.Elixir/SETUP.md

```diff
@@ -0,0 +1,92 @@
+# Workshop 23 - Setup
+
+Please make sure to read this until the end, before running any command !
+
+## Installing necessary dependencies
+
+To run elixir, you will need to install the following versions of
```

Comment:
You said install "the following versions of npm node erlang elix" but you don't mentioned these versions. Propably need more information at this line.